### PR TITLE
Feature/WEB-1906 Add responsive table

### PR DESCRIPTION
**Pull request by BENBEN5959**

### CODE REVIEW

File: src/proxy-page/proxy-page.service.ts [hunk 1]
```diff
@@ -124,6 +125,7 @@ export class ProxyPageService {
     }
     fixSVG(this.config)(this.context);
     fixTableBackground()(this.context);
+    await addTableResponsive()(this.context);
```
Comment (krystiankedra) [inline src/proxy-page/proxy-page.service.ts]: It's not asynchronous function, do You need await ?

File: src/proxy-page/proxy-page.service.ts [hunk 2]
```diff
@@ -187,6 +189,7 @@ export class ProxyPageService {
     fixFrameAllowFullscreen()(this.context);
     fixSVG(this.config)(this.context);
     fixTableBackground()(this.context);
+    await addTableResponsive()(this.context);
```
Comment (krystiankedra) [inline src/proxy-page/proxy-page.service.ts]: It's not asynchronous function, do You need await ?

File: src/proxy-page/proxy-page.service.ts (hunk 1 above)
Comment (BENBEN5959) [reply src/proxy-page/proxy-page.service.ts]: sonarqube suggests me to add await

Comment (krystiankedra) [reply src/proxy-page/proxy-page.service.ts]: it's must be issue on Sonarqube, please remove it, it's unnecessary for all places where You are invoking this step.

**Comment by krystiankedra** [comment]
Please fix this warning in SonarQubee in Your branch

**Pull request merged**